model/registry/process.yaml

```diff
@@ -147,6 +157,15 @@ groups:
           An additional description about the runtime of the process, for example
           a specific vendor customization of the runtime environment.
         examples: 'Eclipse OpenJ9 Eclipse OpenJ9 VM openj9-0.21.0'
+      - id: title
+        type: string
+        stability: experimental
+        brief: >
+          Process title
+        note: >
+          The process title (proctitle), sometimes the same as process name. It can also be different, for
```

Comment:
Could we add more examples here or enhance the description with more specific details? For someone who is not really familiar with this attribute might be hard to interpret.

Reply:
I've rewritten the description and added a few more examples, which hopefully makes it more clear